src/content/en/fundamentals/discovery-and-monetization/_toc.yaml

```diff
@@ -9,4 +9,10 @@ toc:
       path: /web/fundamentals/discovery-and-monetization/social-discovery/
     - title: Monetization with Ads
       path: https://www.google.com/adsense
+    - title: Payment Request API
```

Comment:
Let's call this 'Handling Payments'. The title shouldn't give the impression this is an API reference.